backend/test/akvo/lumen/db/tardis_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns akvo.lumen.db.tardis-test
+  {:functional true}
+  (:require [akvo.lumen.fixtures :refer [*tenant-conn* tenant-conn-fixture]]
+            [clojure.string :as str]
+            [clojure.test :refer :all]
+            [hugsql.core :as hugsql]))
+
+(hugsql/def-db-fns "akvo/lumen/db/tardis_test.sql")
+
+(use-fixtures :each tenant-conn-fixture)
+
+(deftest tardis-with-alter-table
+  (testing "with current data in public schema table, we alter table definition. Functionality keeps the same"
+    (is (= '() (get-data *tenant-conn* {:table-name "data_source"})))
+    (is (= 1 (insert-data-source *tenant-conn* {})))
+    (try
+      (insert-altered-data-source *tenant-conn* {:table-name "data_source" })
+      (assert false)
+      (catch Exception e (is (str/includes? (.getMessage e) "column \"test\" of relation \"data_source\" does not exist") )))
```

Comment:
i think we want to use `thrown-with-msg?` instead of catching the `Exception`
> 
> "(is (thrown-with-msg? c re ...))" does the same thing and also
> tests that the message on the exception matches the regular
> expression re:
>
> (is (thrown-with-msg? ArithmeticException #"Divide by zero"
>                      (/ 1 0)))

https://clojure.github.io/clojure/clojure.test-api.html